app/user/profile/LoanSlider.tsx

```diff
@@ -108,6 +111,47 @@ const LoanSlider = ({ works, loanData }: LoanSliderProps) => {
               </Link>
             )
           })}
+          {works.length < 2 && (
```

Comment:
Would be nice with a comment about the works length in the varoius places of the rendered output.
Maybe before the return statement?